### PR TITLE
fix(table): Fixes an issue with typescript 4 compatibility.

### PR DESCRIPTION
**Pull request by tomheller**
### <strong>Pull Request</strong>

Fixes APM-266777

#### Type of PR

Bugfix

#### Checklist

- [x] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by lukasholzer — DISMISSED** [review]
Maybe reword to a chore.

**Review by bardosmisi — APPROVED** [review]

**Comment by tomheller** [comment]
reworded to chore 👍

**Review by lukasholzer — APPROVED** [review]

**Review by bardosmisi — APPROVED** [review]

**Review by rowa-audil — APPROVED** [review]
